ellama.el

```diff
@@ -193,52 +193,43 @@ in.  Default value is (current-buffer).
 	  (llm-chat-prompt-append-response
 	   ellama--chat-prompt prompt)
 	(setq ellama--chat-prompt (llm-make-simple-chat-prompt prompt)))
-      (unwind-protect
-	  (save-excursion
-	    (let* ((start (make-marker))
-		   (end (make-marker))
-		   (window (selected-window))
-		   (insert-text
-		    (lambda (text)
-		      ;; Erase and insert the new text between the marker cons.
-		      (with-current-buffer (marker-buffer start)
-			(let ((pt (point)))
-			  (save-excursion
-			    (goto-char start)
-			    (delete-region start end)
-			    (insert (funcall filter text))
-			    (fill-region start (point)))
-			  (goto-char pt))
```

Comment:
This shouldn't be necessary given the `save-excursion`.

Reply:
Wrong. See #22

Reply:
I will not work with `save-excursion` without restoring point position.

Reply:
Ah, because the text is being replaced. I've replaced `save-excursion` with manually saving/restoring the point (we don't need both).